code/src/App.js

```diff
@@ -1,9 +1,90 @@
-import React from 'react'
+/*Outer Dependencies*/
+import React, { useState, useEffect } from "react";
 
+/*Local Dependency*/
+import Form from "./components/Form";
+import List from "./components/List";
+
+import { URL } from "./reusable/urls";
+import { URL_HEARTS } from "./reusable/urls";
+
+/* Main App*/
 export const App = () => {
+  /*States*/
+  const [messageList, setMessageList] = useState([]);
+  const [newMessage, setNewMessage] = useState("");
+
+  /*UseEffect*/
+  useEffect(() => {
+    fetchMessageList();
+    console.log("hi!");
+  }, []);
+
+  /*fetch for happy thoughts already posted*/
+  const fetchMessageList = () => {
+    fetch(URL)
+      .then((res) => res.json())
+      .then((messages) => setMessageList(messages))
+      .catch((err) => console.error(err));
+  };
+
+  /*Set the content of new message*/
+  const handleNewMessage = (event) => {
+    setNewMessage(event.target.value);
+  };
+
+  /*Submit of new message*/
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+
+    const options = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ message: newMessage }),
+    };
+
+    /*fetch for object of new posts*/
+    fetch(URL, options)
+      .then((response) => response.json())
+      .then((receivedMessage) =>
+        setMessageList([receivedMessage, ...messageList])
+      )
+      .then(() => setNewMessage(""));
+  };
+
+  /*updating the message-object with +1 heart for each click*/
+  const handleHeartsIncrease = (id) => {
+    const options = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+    };
+
```

Comment:
nice comments above the code. Makes it easier to understand and read the code :)